src/libraries/System.Net.Security/src/System/Net/Security/SslStream.Implementation.cs

```diff
@@ -37,7 +37,9 @@ private enum Framing
         private object _handshakeLock => _sslAuthenticationOptions!;
         private volatile TaskCompletionSource<bool>? _handshakeWaiter;
 
-        private const int FrameOverhead = 32;
+    // 5 byte header + HMAC trailer + padding (if block cipher) 
+    // HMAC: 32 bytes for SHA-256 or 20 bytes for SHA-1 or 16 bytes for the MD5
+        private const int FrameOverhead = 64;
```

Comment:
```suggestion
        // FrameOverhead = 5 byte header + HMAC trailer + padding (if block cipher) 
        // HMAC: 32 bytes for SHA-256 or 20 bytes for SHA-1 or 16 bytes for the MD5
        private const int FrameOverhead = 64;
```

Reply:
Will save this commit to retrigger once the Linq failures one is merged https://github.com/dotnet/runtime/pull/51348